mysql-test/suite/rpl/t/rpl_semi_sync_ack_thread.test

```diff
@@ -240,6 +242,7 @@ SET GLOBAL debug= 'd,dbug.before_get_MASTER_UUID';
 --source include/install_semisync_slave.inc
 
 # Clear the debug point and signal IO thread to resume
+SET DEBUG_SYNC= 'now WAIT_FOR in_get_master_version_and_clock';
```

Comment:
OK, as you wrote on Slack, the merged debug_sync signals first, before waiting. LGTM with two separate SETs